wolox-books/src/app/components/navbar/navbar.component.html

```diff
@@ -1,6 +1,8 @@
 <div class="nav-bar">
 	<div class="content">
 		<img class="logo" src="assets/logo.png" alt="Wolox logo">
+		<img class="shopping-cart" src="assets/cart.png" alt="Shopping cart">
```

Comment:
i would make the shopping cart a separate component, we will add more behaviour to it later